app/views/job_mailer/completion_email.text.erb

```diff
@@ -0,0 +1 @@
+<%=@user.sunet_id%>, your <%=@job.job_type%> job #<%=@job.id%> completed.  View the job details here: "=url_for(@job) will go here".
```

Comment:
We obviously need to change this when we have a job controller and corresponding view here so the route will work

Reply:
talked F2F just add a comment with TODO

Reply:
added in a TODO comment